common/src/main/java/com/wynntils/models/activities/ContentBookQueries.java

```diff
@@ -207,7 +212,8 @@ private String getActiveFilter(ItemStack itemStack) {
 
     private int getFilterChangeDirection(ItemStack itemStack, String targetFilter) {
         StyledText itemName = ItemUtils.getItemName(itemStack);
-        if (!itemName.equals(StyledText.fromString(FILTER_ITEM_TITLE))) return GLFW.GLFW_MOUSE_BUTTON_LEFT;
+        if (!REVERSE_DIRECTION || !itemName.equals(StyledText.fromString(FILTER_ITEM_TITLE)))
+            return GLFW.GLFW_MOUSE_BUTTON_LEFT;
```

Comment:
Brackets :)

Reply:
Sadly spotless can't fix this, but can create it..